addon/components/mox/modal-dialog.hbs

```diff
@@ -6,7 +6,11 @@
 >
   <:default as |titleId|>
     <div class="flex items-center justify-between mb-6">
-      <h1 id={{titleId}} class="text-white font-semibold text-2xl" data-test-modal-dialog-title>
+      <h1
+        id={{titleId}}
+        class="dark:text-white font-semibold text-2xl"
```

Comment:
If you like, you can also explicitly add the `text-gray-800` class here to use the light mode's default text color:

```suggestion
        class="text-gray-800 dark:text-white font-semibold text-2xl"
```